File: HelloWorld.java
import java.util.*;
class HelloWorld
{
  public static void main(String[] args)
  {
    System.out.println("Hello World!!\n");
    System.out.println("Welcome Here! Enter two numbers.\n");
    Scanner sc=new Scanner(System.in);
    String s=sc.nextLine();
    System.out.println("Thanks for your input. \nYour input-"+s);
  }
}

File: HelloWorld.cs
using System;

namespace helloWorld
{
    class HelloWorld
    {
        static void Main(string[] args)
        {
            if(args.Length > 0) {
                if(args[0].Equals("TH")) {
                    SayHelloInThai();
                }
            }
            else {
                Console.WriteLine("Hello World!");
            }
            
        }

        static void SayHelloInThai() {
            Console.WriteLine("Swasdee !!!");
        }
    }
}